module/applications/actor/config/skills-config.mjs

```diff
@@ -0,0 +1,78 @@
+import ProficiencyConfig from "../proficiency-config.mjs";
+import TraitsConfig from "./traits-config.mjs";
+
+/**
+ * Configuration application for actor's skills.
+ */
+export default class SkillsConfig extends TraitsConfig {
+  /** @override */
+  static DEFAULT_OPTIONS = {
+    classes: ["skills"],
+    trait: "skills",
+    actions: {
+      configure: SkillsConfig.#configureSkill
+    }
+  };
+
+  /* -------------------------------------------- */
+
+  /** @override */
+  static PARTS = {
+    traits: {
+      template: "systems/dnd5e/templates/actors/config/skills-config.hbs"
+    }
+  };
+
+  /* -------------------------------------------- */
+  /*  Rendering                                   */
+  /* -------------------------------------------- */
+
+  /** @inheritDoc */
+  async _preparePartContext(partId, context, options) {
+    context = await super._preparePartContext(partId, context, options);
+    context.skills = context.choices.other.children;
+    context.rows = Math.ceil(Object.keys(context.skills).length / 2);
+    return context;
+  }
+
+  /* -------------------------------------------- */
+
+  /** @inheritDoc */
+  _processChoice(data, key, choice, categoryChosen=false) {
+    super._processChoice(data, key, choice, categoryChosen);
+    const skill = data[key];
+    if ( skill ) {
+      choice.value = skill.value;
+      choice.total = this.document.system.skills[key]?.total;
+      choice.tooltip = CONFIG.DND5E.proficiencyLevels[skill.value];
+    }
+  }
+
+  /* -------------------------------------------- */
+  /*  Life-Cycle Handlers                         */
+  /* -------------------------------------------- */
+
+  /** @inheritDoc */
+  _onRender(context, options) {
+    super._onFirstRender(context, options);
+    this.element.querySelectorAll("proficiency-cycle").forEach(e => {
```

Comment:
These don't fire with the normal `_onChangeForm` listeners in `ApplicationV2`?

Reply:
Might not be because `<proficiency-cycle>` doesn't inherit from `AbstractFormInputElement`. I'll mark converting that over for a future PR.